.github/workflows/release_bazel_module.yaml

```diff
@@ -0,0 +1,30 @@
+# Prepare a release specifically for Bazel users, including a pre-built protoc.
+name: Bazel Release
+on:
+  # Can be triggered from the GitHub Actions ui, using the "Run workflow" button on
+  # https://github.com/protocolbuffers/protobuf/actions/workflows/release_bazel_module.yaml
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: git tag that has the protoc release artifact
+        required: true
+        type: string
+permissions:
+  id-token: write
+  attestations: write
+  contents: write
+jobs:
+  release:
+    uses: bazel-contrib/.github/.github/workflows/release_ruleset.yaml@v7.2.3
+    with:
+      release_files: protobuf-*.tar.gz
+      prerelease: false
+      tag_name: ${{ inputs.tag_name || github.ref_name }}
+      bazel_test_command: bazel query @com_google_protobuf//:protoc
```

Comment:
Should we be actually building protoc to test this a little more?

Reply:
Changed to just `true`